### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by ataylorBJSS**
add context settings to avoid lambda timeout
remove stringify of input

### CODE REVIEW

File: src/function/getJournal.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import createResponse from '../util/createResponse';
 import journal from '../service/journal';
 
 export function getJournal(event: any, context: Context, callback: Callback) {
+  context.callbackWaitsForEmptyEventLoop = false;
```
Comment (ammarhaiderbjss) [inline src/function/getJournal.ts:7]: cool 👍

File: src/function/getJournal.ts [hunk 2]
```diff
@@ -14,7 +15,8 @@ export function getJournal(event: any, context: Context, callback: Callback) {
       response = createResponse({ message }, 500);
       callback(err, response);
     } else {
-      response = createResponse({ reply }, 200);
+      const { data: testSlots } = JSON.parse(JSON.parse(reply));
```
Comment (ammarhaiderbjss) [inline src/function/getJournal.ts:18]: how comes we need to parse this twice?

**Review by ammarhaiderbjss — APPROVED** [review]

File: src/function/getJournal.ts (hunk 2 above)
Comment (ataylorBJSS) [reply src/function/getJournal.ts:18]: some weirdness around a string within a string. A single JSON.parse returns a string, the double returns the actual data object. I thought removing the stringify on the set function might help, but didn't seem to affect it.

File: src/function/getJournal.ts (hunk 1 above)
Comment (danielchildsbjss) [reply src/function/getJournal.ts:7]: Why do we do this?

Comment (ataylorBJSS) [reply src/function/getJournal.ts:7]: It's a way of solving the timeout issues we were having after calling redis.

https://stackoverflow.com/questions/41621776/why-does-aws-lambda-function-always-time-out

**Pull request closed without merging**